packages/elements/src/datetime-picker/index.ts

```diff
@@ -1159,13 +1178,40 @@ export class DatetimePicker extends ControlElement implements MultiValue {
     ></ef-time-picker>`;
   }
 
+  /**
+   * Create calendar slot
+   * @param id Calendar identifier
+   * @returns calendarSlots slots that will cascade to calendar
+   */
+  private createCalendarSlots(id: string): HTMLSlotElement[] | null {
```

Comment:
Could we change `id` into `calendarId` for better readability ?

Reply:
sure